examples/generative/corrdiff/generate.py

```diff
@@ -232,9 +232,9 @@ def generate_fn(image_lr):
                         net=net_reg,
                         img_lr=image_lr_patch,
                         seed_batch_size=1,
-                        seeds=[
-                            0,
-                        ],  # Only run regression model once
+                        seeds=list(
+                            range(dist.world_size)
```

Comment:
This will still run the regression model only once per GPU?

Reply:
Yes. In this version, it will distribute a single sample to each GPU. In the previous version with `seeds=[0]`, it resulted in `None` on GPUs that are not `cuda:0`.